scame/formatcheck.py

```diff
@@ -461,6 +461,13 @@ def check(self):
 class AnyTextMixin:
     """Common checks for many checkers."""
 
+    def check_semantic_newline(self):
```

Comment:
this is defined here, but is not 'enabled' in any checkers.

also, since for now we plan to use it only for RST it should be moved into the RST checker.